includes/multi-currency/class-wc-rest-controller.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Class WC_REST_Controller
+ *
+ * @package WooCommerce\Payments\Multi_Currency
+ */
+
+namespace WCPay\Multi_Currency;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for multi-currency.
+ */
+class WC_REST_Controller extends \WC_Payments_REST_Controller {
+
+	/**
+	 * Endpoint path.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'payments/multi-currency';
+
+	/**
+	 * Configure REST API routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/currencies',
+			[
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => [ $this, 'get_store_currencies' ],
+				'permission_callback' => [ $this, 'check_permission' ],
+			]
+		);
+	}
+
+	/**
+	 * Retrieve currencies for the store.
+	 */
+	public function get_store_currencies() {
+		$default    = WC_Payments_Multi_Currency()->get_default_currency();
+		$currencies = [
+			'available' => WC_Payments_Multi_Currency()->get_available_currencies(),
+			'enabled'   => WC_Payments_Multi_Currency()->get_enabled_currencies(),
+			'default'   => [ $default->get_code() => $default ],
```

Comment:
Is it possible for a store to have more than one default currency? If not, what do you think of sending this using the object? Like `'default' => $default`

Reply:
I had done that to be able to loop through and get the data needed, but with your `JsonSerializable` suggestion it makes it so that the default doesn't need to be in an array any more. 😀